Parsers/ASimProcessEvent/Parsers/ASimProcessCreateSentinelOne.yaml

```diff
@@ -0,0 +1,103 @@
+Description: |
+  This ASIM parser supports normalizing SentinelOne logs to the ASIM Process Event normalized schema. SentinelOne events are captured through SentinelOne data connector which ingests SentinelOne server objects such as Threats, Agents, Applications, Activities, Policies, Groups, and more events into Microsoft Sentinel through the REST API.
+ParserName: ASimProcessCreateSentinelOne
+EquivalentBuiltInParser: _Im_ProcessCreate_SentinelOne
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser = (disabled: bool=false) {
+      SentinelOne_CL
+      | where not(disabled) 
+          and event_name_s == "Alerts."
+          and alertInfo_eventType_s == "PROCESSCREATION" 
+      | project-rename
+          DvcId = agentDetectionInfo_uuid_g,
+          EventStartTime = sourceProcessInfo_pidStarttime_t,
+          TargetProcessCommandLine = targetProcessInfo_tgtProcCmdLine_s,
+          TargetProcessId = targetProcessInfo_tgtProcPid_s,
+          TargetProcessName = targetProcessInfo_tgtProcName_s,
+          EventUid = _ResourceId,
+          TargetProcessCreationTime = targetProcessInfo_tgtProcessStartTime_t,
+          DvcHostname = agentDetectionInfo_name_s,
+          ActingProcessName = sourceProcessInfo_name_s,
+          ParentProcessName = sourceParentProcessInfo_name_s,
+          ActingProcessCommandLine = sourceProcessInfo_commandline_s,
+          ActingProcessGuid = sourceProcessInfo_uniqueId_g,
+          ActingProcessSHA1 = sourceProcessInfo_fileHashSha1_s,
+          ParentProcessSHA1 = sourceParentProcessInfo_fileHashSha1_s,
+          ActingProcessSHA256 = sourceProcessInfo_fileHashSha256_s,
+          ParentProcessSHA256 = sourceParentProcessInfo_fileHashSha256_s,
+          DvcOs = agentDetectionInfo_osName_s,
+          DvcOsVersion = agentDetectionInfo_version_s,
+          TargetProcessIntegrityLevel = targetProcessInfo_tgtProcIntegrityLevel_s,
+          EventOriginalType = alertInfo_eventType_s,
+          EventOriginalUid = alertInfo_dvEventId_s
+      | extend
+          ActingProcessId = sourceProcessInfo_pid_s,
+          ActorUsername = coalesce(sourceProcessInfo_user_s, 'N/A'),
+          TargetUsername = coalesce(sourceProcessInfo_user_s, 'N/A'),
+          Hash = coalesce(targetProcessInfo_tgtFileHashSha256_s, targetProcessInfo_tgtFileHashSha1_s),
+          ParentProcessId = sourceProcessInfo_pid_s,
+          TargetProcessSHA1 = targetProcessInfo_tgtFileHashSha1_s,
+          TargetProcessSHA256 = targetProcessInfo_tgtFileHashSha256_s,
+          ParentProcessMD5 = replace_string(sourceParentProcessInfo_fileHashMd5_g, "-", ""),
+          ActingProcessMD5 = replace_string(sourceProcessInfo_fileHashMd5_g, "-", ""),
+          EventSeverity = iff(ruleInfo_severity_s == "Critical", "High", ruleInfo_severity_s)
+      | extend
+          EventCount = int(1),
+          EventProduct = "SentinelOne",
+          EventResult = "Success",
```

Comment:
Is there EventOriginalResult value coming from source?

Reply:
No